llvm/utils/repo/Makefile

```diff
@@ -114,26 +123,46 @@ libcxx.a: libcxxabi.a
 		-D CMAKE_TOOLCHAIN_FILE=$(TOOLCHAIN_FILE)                     \
 		-D CMAKE_CXX_COMPILER=$(LLVM_BUILD)/bin/clang++               \
 		-D CMAKE_C_COMPILER=$(LLVM_BUILD)/bin/clang                   \
+		-D utils_dir=$(LLVM_REPO)/llvm/utils/repo                     \
 		-D CMAKE_INSTALL_PREFIX=$(LLVM)                               \
 		-D musl_install=$(MUSL)                                       \
 		../libcxx &&                                                  \
 	make -j 8 &&                                                          \
 	make install
 
+$(LLVM)/lib/libunwind.db: libunwind.a
+	cp $(LLVM_REPO)/build_libunwind/libunwind.db $@
+
+$(LLVM)/lib/linux/libcompiler-rt.db: libcompiler-rt.a
+	cp $(LLVM_REPO)/build_libcompiler-rt/libcompiler-rt.db $@
+
+$(LLVM)/lib/libc++abi.db: libcxxabi.a
+	cp $(LLVM_REPO)/build_libcxxabi/libc++abi.db $@
+
+$(LLVM)/lib/libc++.db: libcxx.a
+	cp $(LLVM_REPO)/build_libcxx/libc++.db $@
+
+install-libs: libunwind.a \
+	libcompiler-rt.a \
+	libcxxabi.a \
+	libcxx.a
+
+install-repo-libs-dbs: $(LLVM)/lib/libunwind.db \
+	$(LLVM)/lib/linux/libcompiler-rt.db \
+	$(LLVM)/lib/libc++abi.db \
+	$(LLVM)/lib/libc++.db
+
 .PHONY: all
-all :
-	$(MAKE) libunwind.a \
-		libcompiler-rt.a \
-		libcxxabi.a \
-		libcxx.a
+all:
+ifeq ($(FULL_REPO),yes)
+	$(MAKE) install-repo-libs-dbs
+else
+	$(MAKE) install-libs

```

Comment:
Shouldn’t this also happen for the “full repo” case?

Reply:
Since `libunwind.db` is dependent on `libunwind.a`, `libcompiler-rt.db` is dependent on `libcompiler-rt.a`, `libc++abi.db` is dependent on `libcxxabi.a`, `libc++.db` is dependent on `libcxx.a`, `install-repo-libs-dbs` not only installed repo databases but also did the same task of `install-libs`. Therefore, I didn’t put `install-libs` in full_repo case.